webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Skeleton from 'react-loading-skeleton';
+
+import { useQuery } from '@apollo/client';
+import variableOverrides from '../../../../graphql/queries/variableOverrides.gql';
+import AnsibleVariableOverridesTable from './AnsibleVariableOverridesTable';
+
+import { encodeId } from '../../../../globalIdHelper';
+import { reorderVariables } from './AnsibleVariableOverridesHelper';
+import './AnsibleVariableOverrides.scss';
+
+const AnsibleVariableOverrides = ({ hostAttrs }) => {
+  const { id } = hostAttrs;
+
+  const { loading, data, error } = useQuery(variableOverrides, {
+    variables: { hostId: id, id: encodeId('Host', id) },
```

Comment:
It might be used instead of `hostId`, but real id is still needed to construct a global id, so the whole component is rendered only after the id is available.